tests/test_optimizer.py

```diff
@@ -297,12 +297,49 @@ def run_worker(index: int, batch_size: int, period: float, **kwargs):
 
 
 @pytest.mark.forked
+@pytest.mark.parametrize(
+    "use_local_updates, delay_state_averaging, delay_optimizer_step, delay_grad_averaging, reuse_grad_buffers",
+    [
+        (False, False, False, False, False),
+        (False, True, False, False, False),
+        (False, True, True, True, False),
+        (False, False, False, False, True),
+        (False, True, True, True, True),
+        (False, True, True, False, True),
+        (True, False, False, False, False),
+        (
+            True,
+            True,
+            False,
+            False,
+            False,
+        ),
```

Comment:
To make formatting more consistent, you can surround the entire block of cases above with `# fmt: off` and `# fmt: on`

Reply:
good point, thx